plugins/ocp/ocp-print-json.c

```diff
@@ -569,10 +569,10 @@ static void json_c9_log(struct telemetry_str_log_format *log_data, __u8 *log_dat
 	json_object_add_value_int(root, "Log Page Version",
 				  le16_to_cpu(log_data->log_page_version));
 
-	memset((__u8 *)res, 0, 15);
+	memset((__u8 *)res, 0, 48);
 	for (j = 0; j < 15; j++)
 		res += sprintf(res, "%d", log_data->reserved1[j]);
-	json_object_add_value_string(root, "Reserved", res_arr);
+	json_object_add_value_string(root, "Reserved[15:01]", res_arr);
```

Comment:
Printing the reserved fields in the json output is problematic. IMO. I kind of get it when printing on the console. The JSON output is for machines to parse. So updating the field name is a regression and when we add new fields the output might change as well. I think we should just not output these at all.

Obviously, this reserved field is already here and we should not break parsers. So I'd say just leave it as it is, don't rename it.

Reply:
I get your point. Since, this usage of reserved key had corrupted data added in its value field. 
However, the intention is not to break the parsers already available and machines don't care about reserved value. I will leave it as its , without renaming

Reply:
One thing we could do is that we just append the 'reserved' field with no data at all. I don't expect that `res_arr` should be exposed.

Reply:
Oh, I just retained reserved and it anyway has non key value (just zero). 
Updated the pull request with the latest code. please do check.